CHANGELOG.md

```diff
@@ -60,17 +60,17 @@
 - Added `Import-ContentLibraryItem` cmdlet to support adding files to a vSphere Content Library.
 - Added `Export-PcaJsonSpec` cmdlet to generate a JSON specification file for Private Cloud Automation.
 - Added `Invoke-PcaDeployment` cmdlet to perform an end-to-end deployment of Private Cloud Automation.
-- Added `Request-AslcmBundle` cmdlet to download the VMware Aria Suite Lifecycle bundle in SDDC Manager.
-- Added `New-AslcmDeployment` cmdlet to perform the deployment of VMware Aria Suite Lifecycle using SDDC Manager.
-- Added `Install-AslcmCertificate` cmdlet to install a signed certificate for VMware Aria Suite Lifecycle using SDDC Manager.
+- Added `Request-VrslcmBundle` cmdlet to download the VMware Aria Suite Lifecycle bundle in SDDC Manager.
+- Added `New-VrslcmDeployment` cmdlet to perform the deployment of VMware Aria Suite Lifecycle using SDDC Manager.
+- Added `Install-VrslcmCertificate` cmdlet to install a signed certificate for VMware Aria Suite Lifecycle using SDDC Manager.
```

Comment:
Please remove this change, the new functions have been created with Aslcm name deliberately